packages/ui/src/assets/css/checkbox.css

```diff
@@ -1,9 +1,14 @@
+:root {
+  --input-background-color: var(--dz-primary-color);
+  --error-message-color: var(--dz-danger-color);
+}
+
```

Comment:
Instead of doing this in every css file, would it be better to do it from the vars.css file? We have the variables defined there, we just need to point them to standard colors eg In vars.css file: 
```
--input-background-color: var(--dz-primary-color);`
```